<ui version="4.0" >
 <class>OnlineToaster</class>
 <widget class="QWidget" name="OnlineToaster" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>221</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="windowFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <property name="horizontalSpacing" >
       <number>0</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="2" >
         <widget class="QPushButton" name="closeButton" >
          <property name="minimumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="icon" >
           <iconset resource="../images.qrc" >:/images/close-down.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>225</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>61</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap resource="../images.qrc" >:/images/kdmconfig.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>111</width>
            <height>71</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>137</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="messageButton" >
          <property name="text" >
           <string>Peer Online</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>136</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>368</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>messageButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
